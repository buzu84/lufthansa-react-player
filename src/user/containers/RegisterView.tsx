import React from 'react'
import { useForm } from "react-hook-form";

interface Props {

}

export const RegisterView = (props: Props) => {

    return (
        <div>
            <div className="form-group">
                <label htmlFor="username">Username:</label>
                <input id="username" type="text" className="form-control" />
            </div>

            <div className="form-group">
                <label htmlFor="password">Password:</label>
                <input id="password" type="text" className="form-control" />
            </div>

            <div className="form-group">
                <button className="btn btn-danger">Cancel</button>
                <button className="btn btn-sucess">Register</button>
            </div>
        </div>
    )
}
