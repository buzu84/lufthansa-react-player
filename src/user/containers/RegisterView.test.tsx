import React from 'react'

import renderer from 'react-test-renderer';
import { RegisterView } from './RegisterView';

describe('RegisterView', () => {
    
    test('Should render form', () => {
        const tree = renderer
            .create(<RegisterView />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})
