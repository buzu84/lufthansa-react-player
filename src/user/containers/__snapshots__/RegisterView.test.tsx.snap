// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterView Should render form 1`] = `
<div>
  <div
    className="form-group"
  >
    <label
      htmlFor="username"
    >
      Username:
    </label>
    <input
      className="form-control"
      id="username"
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="password"
    >
      Password:
    </label>
    <input
      className="form-control"
      id="password"
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <button
      className="btn btn-danger"
    >
      Cancel
    </button>
    <button
      className="btn btn-sucess"
    >
      Register
    </button>
  </div>
</div>
`;
